website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -88,7 +88,7 @@ By supplying the necessary attributes under a `vcs-repository` object, you can c
 | `data.attributes.vcs-repo.identifier`         | string  |           | A reference to your VCS repository in the format :org/:repo where :org and :repo refer to the organization and repository in your VCS provider. The format for Azure DevOps is `:org/:project/_git/:repo`.                                                                                                                                                                                                                                                                                                      |
 | `data.attributes.vcs-repo.tags-regex`         | string  | (nothing) | A regular expression used to match Git tags. Terraform Cloud triggers a run when this value is present and a VCS event occurs that contains a matching Git tag for the regular expression.                                                                                                                                                                                                                                                                                                                      |
 | `data.attributes.working-directory`           | string  | (nothing) | A relative path that Terraform will execute within. This defaults to the root of your repository and is typically set to a subdirectory matching the environment when multiple environments exist within the same repository.                                                                                                                                                                                                                                                                                   |
-| `data.attributes.drift_detection`             | boolean | `false` | Whether or not Terraform Cloud performs drift detection assessments for the workspace. Only available for Business tier organizations, in workspaces running Terraform version 0.15.4+ and operating in [Remote execution mode](/cloud-docs/workspaces/settings#execution-mode). Not available in Terraform Enterprise. |
+| `data.attributes.assessments-enabled`         | boolean | `false` | (previously `drift-detection`) Whether or not Terraform Cloud performs health assessments for the workspace. May be overriden by the organization setting `enforce-assessments`. Only available for Business tier organizations, in workspaces running Terraform version 0.15.4+ and operating in [Remote execution mode](/cloud-docs/workspaces/settings#execution-mode). Not available in Terraform Enterprise. |
```

Comment:
I'm trying to do complete sentences. Do you hate it? if you hate it put it back, but keep my spelling fix :-) hmmm, I also know what we're trying to do is warn folks that this used to be something else, but statements like "previously" can get stale. Like, when?!  I think we should remove that and just loudly tell folks everywhere else (blogs, idk other communications) that this happened.

The one place I think it makes sense to say "used to be drift detection" is on that page, because we renamed it, and I think it'd be weird to not mention that there. Push back on me if you completely disagree, but I just want to limit our use of things like this to only where they are very necessary.

```suggestion
| `data.attributes.assessments-enabled`         | boolean | `false` | Whether or not Terraform Cloud performs health assessments for the workspace. This property can be overridden by the organization setting `enforce-assessments`. This property is only available for Business tier organizations, in workspaces running Terraform version 0.15.4+ and operating in [Remote execution mode](/cloud-docs/workspaces/settings#execution-mode). This property is not available in Terraform Enterprise. |
```